### ISSUE TITLE
understanding transcriptome information...

### ISSUE THREAD

**Issue by tmanzurc** [sr320/austral#13]
I really get to understand more what we were doing with all our command lines when I was able to see the GOslim terms and visualize them for example in a graph.

**Comment by sr320** [comment]
You are correct we do not always need to use the command line, some times I was just using it to show you examples of all the things it can do for you.  

It is much easier to make a mistake in programs such as Excel- as compared to the command-line
